package.json

```diff
@@ -4,7 +4,7 @@
   "description": "chat",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "mocha --compilers js:babel-core/register --require ./app/tests/test_helper.js './app/tests/**/*.@(js|jsx)'",
```

Comment:
Also could be nice to add smth like: `"test-client:watch": "npm run test-client -- --watch",`